src/utils/enumsStructs.h

```diff
@@ -0,0 +1,220 @@
+//
+// Created by daniel on 29.05.24.
+//
+
+#pragma once
+
+#include <array>
+
+namespace enumsStructs {
+    /**
+     * Enum to specify the side that can be chosen in some methods.
+     */
+    enum class Side {
+        front, right, back, left, top, bottom
```

Comment:
Slightly odd ordering here.
Something like 
```suggestion
        front, back, right, left, top, bottom
```
might be beneficial if you algorithmically do things on sides.. 
Probably irrelevant at the  current stage though.